docs/tutorials/sidebar-tutorial/plugin-sidebar-0.md

```diff
@@ -0,0 +1,12 @@
+# Creating a Sidebar for Your Plugin
```

Comment:
This file was previously: `docs/designers-developers/developers/tutorials/sidebar-tutorial/plugin-sidebar-0.md`. It looks like GitHub didn't pick up this rename.